client/src/grade/Grade.js

```diff
@@ -8,75 +9,143 @@ class Grade extends React.Component{
+        }).then(res => {
+    		//const evalInfo = res.data;
+    		this.setState({ evalInfo : res.data});
+    	}).then ( res =>{
+            console.log(res)
+          }).catch((error) =>{
+              if(error.response){
+                console.log(error.response.data);
+              } else if (error.request){
+                  console.log(error.request); 
+              }else {
+                  console.log(error.message);
+              }
+          })         
     }
 
     updateSlider = (event) => {
         this.handleSliderChange(event);
-        document.getElementById(event.target.id).innerHTML=event.target.value;
-        this.setState({ [event.target.name]: event.target.value });
+        document.getElementById(event.target.id).innerHTML=event.target.value
+    }
+
+    filler(i){
+        if(!this.state.loading){
+            return this.state.evaluation[i].grade
+        } else {
+            return i+1
+        }
     }
 
     tableBody(){
         return (
-            this.table = this.state.info.map((data, i) => 
+            this.table = this.state.evalInfo.map((data, i) => 
                 <tr>
                     <td>
-                        <b>{data.q_num}) </b>{data.q_resp}<br/>
+                        <b>{(i+1)}) </b>
+                        {data.prompt_text}<br/><br/>
+                        {data.question_text}<br/><br/>
+                        {data.response_value}<br/><br/>
                         <table className="gradeTable">
                             <tr>
-                                Clarity: <t id={"Clarity:"+data.q_num}>{this.state.rubric[i].q_clarity}  </t>
-                                    <input type="range" min="1" max="10" value="5" className="slider" 
-                                        id={"Clarity:"+data.q_num} 
-                                        value={this.value} 
-                                        onChange={this.updateSlider}/>
-                            </tr>
-                            <tr>
-                                Understanding: <t id={"Understand"+data.q_num}>{this.state.rubric[i].q_undstd}</t>
-                                    <input type="range" min="1" max="10" value="5" className="slider" 
-                                        id={"Understand"+data.q_num} 
-                                        value={this.value} 
-                                        onChange={this.updateSlider}/>
-                            </tr> 
-                            <tr>
-                                Accuracy: <t id={"Accuracy"+data.q_num}>{this.state.rubric[i].q_acc}</t>
-                                    <input type="range" min="1" max="10" value="5" className="slider" 
-                                        id={"Accuracy"+data.q_num} 
+                                    Grade: <t id={(i+1)}>{this.filler(i)}</t>
+                                    <input type="range" min="1" max="100" value="5" className="slider" 
```

Comment:
This input needs a unique name which you can get from `eval_id`. This means adding `name={'slider' + data.eval_id}`. then value should instead be 'value={this.state['slider' + data.eval_id]}`. This will make it function dynamically like your meaning it to.